### PR TITLE
@types/auth0: add UserManager member to ManagementClient + getUserOrg…

### PR DESCRIPTION
**Pull request by Alex-Yakubovsky**
…anizations to UserManager

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
    *  [UsersManager.getUserOrganizations](https://auth0.github.io/node-auth0/UsersManager.html#getUserOrganizations)
    * [ManagementClient.users](https://auth0.github.io/node-auth0/management_index.js.html#line181)

### CODE REVIEW

File: types/auth0/index.d.ts [hunk 1]
```diff
@@ -2171,4 +2172,7 @@ export class UsersManager<A = AppMetadata, U = UserMetadata> {
 
     impersonate(userId: string, settings: ImpersonateSettingOptions): Promise<any>;
     impersonate(userId: string, settings: ImpersonateSettingOptions, cb: (err: Error, data: any) => void): void;
+
+    getUserOrganizations(params: ObjectWithId): Promise<Organization[]>;
+    getUserOrganizations(params: ObjectWithId, cb: (err: Error, orgs: Organization[]) => void): void;
```
Comment (peterblazejewicz) [inline types/auth0/index.d.ts:2177]:
```suggestion
    getUserOrganizations(data: ObjectWithId): Promise<Organization[]>;
    getUserOrganizations(data: ObjectWithId, cb: (err: Error, orgs: Organization[]) => void): void;
```
stick to the docs, it's a singular param

Comment (Alex-Yakubovsky) [reply types/auth0/index.d.ts]: oops, fixed

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Alex-Yakubovsky thanks!

**Comment by Alex-Yakubovsky** [comment]
Ready to merge

**Pull request merged**